src/app/profile/profile/profile.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="wrapper">
+  <p>profile works!</p>
+  <button mat-fab routerLink="/lots/new">Click me!</button>
```

Comment:
this link goes to `http://localhost:4200/lots/new` while it should go to `http://localhost:4200/profile/lots/new`

Reply:
done